src/ui/controls/list.cpp

```diff
@@ -17,8 +17,9 @@
  * along with this program. If not, see http://gnu.org/licenses
  */
 
-
 #include "ui/controls/list.h"
+#include "ui/controls/button.h"
```

Comment:
`Expected empty line between include directives` (the main .h related to .cpp should be separated)
`Broken alphabetical ordering, expected 'common/make_unique.h', not 'ui/controls/button.h'`